netbeans/CheckerFrameworkPlugin/release/checkerstrings.properties

```diff
@@ -0,0 +1,18 @@
+org.checkerframework.checker.nullness.NullnessChecker=Nullness Checker
+org.checkerframework.checker.interning.InterningChecker=Interning Checker
+org.checkerframework.checker.lock.LockChecker=Lock Checker
+org.checkerframework.checker.fenum.FenumChecker=Fenum Checker
+TaintingChecker=Tainting Checker
```

Comment:
The short names actually work for all checkers, if one uses the Checker Framework javac script.
For consistency, I would use the fully-qualified name for all checkers.